loaders/src/glTF/2.0/Extensions/KHR_materials_translucency.ts

```diff
@@ -86,7 +86,7 @@ export class KHR_materials_translucency implements IGLTFLoaderExtension {
             return this._loader.loadTextureInfoAsync(`${context}/translucencyTexture`, extension.translucencyTexture)
                 .then((texture: BaseTexture) => {
                     pbrMaterial.subSurface.thicknessTexture = texture;
-                    pbrMaterial.subSurface.useMaskFromThicknessTextureGltf = true;
+                    pbrMaterial.subSurface.useGltfStyleThicknessTexture = true;
```

Comment:
Is this naming change a breaking change or was `useMaskFromThicknessTextureGltf` new in 5.0?

Reply:
Yes, I guess it would be. However, I can't imagine that anyone was using this except for the extension loader.